src/__mock__/addandremove.test.js

```diff
@@ -0,0 +1,10 @@
+describe('should addandremove a task', () => {
+  test('checking for task removal', () => {
+    const arr = [2, 3, 5];
+    expect(arr.splice(1, 2)).toHaveLength(2);
+  });
+  test('checking for task addition', () => {
+    const arr = [3];
+    expect(arr.length).toBe(1);
+  });
+});
```

Comment:
- [ ] Based on the requirements you are required to write tests for the **specific** methods responsible for adding and removing a task, but that is not the case here. You need to implement this addition and removal using your custom methods, not inbuilt JS methods.

- [ ] Kindly mock the local storage or DOM and ensure the add and remove methods add and remove tasks from them.

Reply:
thanks, but please, could you kindly explain further and maybe suggest how to approach these changes @DammyShittu